ts/Extensions/Breadcrumbs.ts

```diff
@@ -265,6 +265,15 @@ class Breadcrumbs {
          */
         relativeTo: 'plotBox',
 
+        /**
+         * Whether to reverse the order of buttons. This is common in Arabic
+         * and Hebrew.
+         *
+         * @type       {boolean}
```

Comment:
A visual sample would also be nice. It would double as a test and a demo. Create a button with `class="autocompare"` to perform the drilldown. Search for "autocompare" in the samples for inspiration.